pages/in-vitro-systems/[id].js

```diff
@@ -154,9 +154,8 @@ export async function getServerSideProps({ params, req, query }) {
     `/in-vitro-systems/${params.id}/`
   );
   if (FetchRequest.isResponseSuccess(inVitroSystem)) {
-    const biosampleTerm = inVitroSystem.biosample_term
-      ? await request.getObject(inVitroSystem.biosample_term["@id"], null)
-      : null;
+    const sampleTerms =
+      inVitroSystem.sample_terms?.length > 0 ? inVitroSystem.sample_terms : [];
```

Comment:
With the above change, you don’t need to extract `invitroSystem.sample_terms` and `invitroSystem.sources` into their own variables, and you don’t need to return them in `props`.

Reply:
Done